### PR TITLE
:bug: Walkaround for OCPBUGS-38680

### PR DESCRIPTION
**Pull request by cardil**
Selecting Serverless as resource type, as it ceased to be a default one for ODC 4.17+.

Walk around for https://issues.redhat.com/browse/OCPBUGS-38680

## Proposed Changes

- :bug: Selecting Serverless as resource type, as it ceased to be a default one for ODC 4.17+

/kind bug

### CODE REVIEW

File: knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml [hunk 1]
```diff
@@ -1153,22 +1153,15 @@ metadata:
   labels:
     app.kubernetes.io/version: v1.14
     knative.dev/crd-install: "true"
-  name: consumergroups.internal.kafka.eventing.knative.dev
+  name: consumers.internal.kafka.eventing.knative.dev
```
Comment (mgencur) [inline knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml:1156]: Hmm, why are the changes here? It's unrelated to UI.
Maybe you need to rebase?

**Review by mgencur — COMMENTED** [review]
The changes in showcase.js look good. But there are some other changes which shouldn't be there.

File: knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml (hunk 1 above)
Comment (cardil) [reply knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml:1156]: I did a rebase, already.

Yes, those changes aren't expected. I'll try rebasing one more.

Comment (mgencur) [reply knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml:1156]: This needs to be merged: https://github.com/openshift-knative/serverless-operator/pull/2813/files

Comment (cardil) [reply knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml]: Done in 203fc8d231982c3c2bf194c3cecf589d1b1c082e

**Comment by mgencur** [comment]
/lgtm
/hold
Hold for 412-ui-e2e-aws-412 to pass.

**Comment by cardil** [comment]
This change isn't needed. The tests work well on 4.17 even without it.

See https://issues.redhat.com/browse/OCPBUGS-38680?focusedId=25409019&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25409019 for details.

**Pull request closed without merging**